### PR TITLE
Hm 13

### PR DESCRIPTION
**Pull request by SlavaKoshun**

### CODE REVIEW

File: src/homework13/ex1/Printable.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package ex1;
+
+public interface Printable {
```
Comment (AndrewSkoba) [inline src/homework13/ex1/Printable.java:3]: Better to add @FunctionalInterface here

File: src/homework13/ex1/Printable.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+
+    void print();
+
+    Printable printableLambda = () -> System.out.println("Printing...");
```
Comment (AndrewSkoba) [inline src/homework13/ex1/Printable.java]: Move it to some other place. For example to Runner class

File: src/homework13/ex1/Runner.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+public class Runner {
+  public static void main(String[] args){
+
+      Printable[] printables = {new Book("Math"),new Magazine("Moto"),new Book("Java Core"), new Magazine("New Car")};
+
+      System.out.println("------------------");
+
+      for (Printable printable : printables) {
+          printable.print();
```
Comment (AndrewSkoba) [inline src/homework13/ex1/Runner.java:22]: Method reference here

File: src/homework13/ex1/Magazine.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    }
+
+    @Override
+    public void  print() {
+        System.out.println("Magazine - " + name);
+    }
+
+    public static void printMagazines(Printable @NotNull [] printable) {
+        System.out.println("Printing Magazine: ");
+        for (Printable num : printable) {
```
Comment (AndrewSkoba) [inline src/homework13/ex1/Magazine.java]: Try to rewrite with Java8.
There is no in HW)

File: src/homework13/ex2/Sorting.java [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+         */
+        int averageAge = (int)list.stream().mapToInt(User:: getAge).average().getAsDouble();
+        System.out.println("Середній вік: " + averageAge);
+        System.out.println("--------------");
+
+        /**
+         * <p>Список User які мають вік > 18.</p>
+         */
+        List<User> ageVerification = list.stream()
+                .filter(User -> User.getAge() > 18)
```
Comment (AndrewSkoba) [inline src/homework13/ex2/Sorting.java:46]: naming strategy the same as for local/global variables. 
variables - starting from small latter

Comment (SlavaKoshun) [reply src/homework13/ex2/Sorting.java:46]: не розумію про яку ти змінну) ніби всі з малої)

**Review by AndrewSkoba — APPROVED** [review]

**Pull request merged**